base/asyncmap.jl

```diff
@@ -22,17 +23,17 @@ end
 
 function AsyncCollector(f, results, c...; ntasks=0)
     if ntasks == 0
-        ntasks = 100
+        ntasks = max(nworkers(), 100)
```

Comment:
thanks @amitmurthy !  per the commands in that link, the number of local temporary ports is ~30k.  so given the `:all_to_all` default topology, i shouldn't be able to add more than ~170 workers.  seemingly 100 was close enough to the limit.  please let me know if there is anything i can do to help get `:master_slave` working, or to have `:all_to_all` fail more gracefully.